utils/s3/client.hh

```diff
@@ -51,6 +51,15 @@ public:
         std::time_t last_modified;
     };
     future<stats> get_object_stats(sstring object_name);
+    struct Tag {
```

Comment:
Why `Tag` but not `tag`?
I would also put it at namespace scope next to the `s3::range` struct in the same header

Reply:
i wanted to match the names in its API definition, see https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketTagging.html. but `tag` is fine by me, if we want to use lower-cased names for the data-only structs.